_pages/en/it-picture-medium-term.md

```diff
@@ -26,124 +26,131 @@ The foundational pieces for "Government as a service" vision are available and i
 
-Overall employees working in IT at ESDC are happier, more productive and more up to date on technology, making them better able to support the needs of the department so that it can deliver more efficient services to Canadians.
+Overall employees working in IT at ESDC are happier, more productive and more up to date on technology, making them better able to support the evolving needs of the department so that it can deliver more efficient services to Canadians.
 
 #### Process
 
-- https://cloud.google.com/solutions/devops/devops-process-team-experimentation
-- https://cloud.google.com/solutions/devops/devops-process-streamlining-change-approval
-- https://cloud.google.com/solutions/devops/devops-process-customer-feedback
-- https://cloud.google.com/solutions/devops/devops-process-work-visibility-in-value-stream
-- https://cloud.google.com/solutions/devops/devops-process-working-in-small-batches
+- [https://cloud.google.com/solutions/devops/devops-process-team-experimentation](https://cloud.google.com/solutions/devops/devops-process-team-experimentation)
+- [https://cloud.google.com/solutions/devops/devops-process-streamlining-change-approval](https://cloud.google.com/solutions/devops/devops-process-streamlining-change-approval)
+- [https://cloud.google.com/solutions/devops/devops-process-customer-feedback](https://cloud.google.com/solutions/devops/devops-process-customer-feedback)
+- [https://cloud.google.com/solutions/devops/devops-process-work-visibility-in-value-stream](https://cloud.google.com/solutions/devops/devops-process-work-visibility-in-value-stream)
+- [https://cloud.google.com/solutions/devops/devops-process-working-in-small-batches](https://cloud.google.com/solutions/devops/devops-process-working-in-small-batches)
 
 Processes empower cross-functional teams and allow them to work on new ideas in pursuit of business goals that solve important problems.
-Teams are provided with information and context letting them make informed decisions about the right work to do.
-Information is open by default, reducing the burden on employees to request access to it.
-Organizational outcomes are measured and provide information critical to making the best decisions.
-Hackathons are held twice a year by IT.
+Information is treated as a corporate asset.
+It is open by default, easily searchable, secure and not duplicated.
+Teams are provided with up-to-date information and context enabling them to make informed decisions about the right work to do.
+Organizational outcomes are measured and also provide information critical to making the best decisions.
+Hackathons are held at least twice a year by IT and are open to internal and relevant external audiences (e.g. other governments, academia).
 
 Teams work using Agile methodologies to iterate on and continually improve solutions and services.
 Clients are included from the start of projects and can submit regular feedback.
```

Comment:
Should we change the word "clients" to business partners or something?

Reply:
Make sense to me. For those more knowledgeable about Agile than myself, is that the most common branding?

Reply:
I believe so but we can also change again later ;)